src/Symfony/Component/RemoteEvent/Event.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\RemoteEvent;
+
+class Event
```

Comment:
`RemoteEvent` instead? Although it's redundant with the namespace, I think that having multiple `Event` classes hurts DX even more

Reply:
No experimental flag also, is it on purpose?

Reply:
Done both.